storage/gcp/bucket/main.tf

```diff
@@ -0,0 +1,118 @@
+data "google_client_config" "current" {}
+
+resource "google_storage_bucket" "bucket_creation" {
+  name                        = var.bucket_name
+  location                    = data.google_client_config.current.region
```

Comment:
@lzianekhodja I don't remember, on what did we settled for the region? explicit variable? or from the provider config?

Reply:
We use the region and project ID provided in the google provider

Reply:
Actually, buckets can be deployed in multiple regions: https://cloud.google.com/storage/docs/locations

I would say you could have an argument to specify the location, and just use the local region if no location was given.

Reply:
Add variable alternative location (by default null) that replace the project location if not null